README.md

```diff
@@ -40,9 +53,10 @@ Default path to config file is
   ```
 
 ## Driver Installation
-After the above Prerequisites are met, Use the standard golang install process:
+After the above Prerequisites are met, clone repository and use the Makefile:
   ```
-  go get -u github.com/nexenta/nexenta-docker-driver/...
+  git clone https://github.com/nexenta/nexenta-docker-driver
```

Comment:
If 'go get' installs all the required modules, why did you replace 'go get' with 'git clone'? :)